packages/legacy-workbench/src/js/angular/models/ttyg/agent-form.js

```diff
@@ -108,6 +128,7 @@ export class AgentFormModel {
             name: this._name,
             repositoryId: this._repositoryId,
             model: this._model,
+            contextSize: this._contextSize,
```

Comment:
Shouldn't api field be added to the payload as well or it's  not part of the requets model?